docs/dev-practices/code_review.md

```diff
@@ -0,0 +1,34 @@
+## Code Review
+
+Pull requests should be titled in the format of `#issue_number: Descriptive name ideally matching ticket name - [sandbox]`
+Pull requests including a migration should be suffixed with ` - MIGRATION`
+
+After creating a pull request, pull request submitters should:
+- Add at least 2 developers as PR reviewers (only 1 will need to approve).
+- Message on Slack or in standup to notify the team that a PR is ready for review.
+- If any model was updated to modify/add/delete columns, run makemigrations and commit the associated migrations file.
+
+## Pull request approvals
+Code changes on user-facing features (excluding content updates) require approval from at least one developer and one designer.
+All other changes require a single approving review.
+
+The submitter is responsible for merging their PR unless the approver is given explcit permission. Similarly, do not commit to another person's branch unless given explicit permission.
+
+Bias towards approving i.e. "good to merge once X is fixed" rather than blocking until X is fixed, requiring an additional review.
+
+## Pull Requests for User-facing changes
+When making user-facing changes, test that your changes work on multiple browsers including Chrome, Microsoft Edge, Firefox, and Safari.
+
+Add new pages to the .pa11yci file so they are included in our automated accessibility testing.
+
+## Other Pull request norms
+- Keep pull requests as small as possible. This makes them easier to review and track changes.
+- Write descriptive pull requests. This is not only something that makes it easier to review, but is a great source of documentation. 
+
+[comment]: The Coding standards section will be moved to a new code standards file in #2898. For now we're simply moving PR template content into the code review document for consolidation
```

Comment:
**blocking question** Okay this is what I am unsure about, I don't think we should have code standards in a separate file as then no one will look at it when doing a PR review. At least that is what I guess will happen. The context switch usually means people don't end up going to the other reference, or if they do, they go once. What are your thoughts?

Reply:
Hm I didn't consider that. Are you referring to having code standards in a separate file from code_review.md or the PR template overall? 

For the PR template: I thought the opposite and thought separating our general code standards and the PR approval steps would let us condense our PR template into its bare necessity checks to approve while keeping our code standards somewhere else. I observed people didn't seem to check off the code standards boxes on our PRs anyway so that was mainly why I separated the two. 

For code_review.md: I don't have strong opinions on whether we separate our code review practices and coding standards into separate files. We had a past engineering huddle when we considered the possibility the coding standards we wanted to document would get more complex, and I mentioned we could move coding standards into a separate file if that's the case. Happy to keep coding standards and code review standards in the same file if that's unlikely

Reply:
Definitely meant for the code_review.md file! You're right having them out of the template is a good thing. lets just remove this line for now and then just bring up in eng huddle and do a quick vote on if people want code standards as it's own md vs in the code_review.md.

Reply:
Makes sense! I thought it was code_review but wanted to make sure I didn't misinterpret, thank you! I'll take it out and we can also discuss it once the code standards docs ticket gets worked on